ls1intum/Artemis
# [Feature] Redirect to example submission board after the creation of a text/modelling exercise.

### Checklist

### Description

After the instructor creates a text/modeling exercise the system should redirect to the example submissions dashboard (created in the previous PR: https://github.com/ls1intum/Artemis/pull/3443)

The motivation of this PR is to make it more obvious and easy for instructors to create example submissions.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Select a course
4. Click exercises
5. Crete a text/modeling exercise
6. Upon creation you should be redirected to the example submission dashboard

### GIF
![DEMO (3)](https://user-images.githubusercontent.com/51077603/121603518-2eb57780-ca49-11eb-966f-33524c36829d.gif)
